BUILDING.md

```diff
@@ -59,6 +59,13 @@ note1 - The gcc4.8-libs package needs to be installed, because node
   by Joyent. SmartOS images >= 16.4 are not supported because
   GCC 4.8 runtime libraries are not available in their pkgsrc repository
 
+note2 - Running Node.js on Windows is officially supported using Cmd. Running Node.js
```

Comment:
I think I understand the place this is coming from, but it's a way to complicated scenario to try to boil down to a *Note*...
For instance there's also `PowerShell` which is supported, `WSL` is a complete different animal (and is it `Ubuntu` or `OpenSuse`). Also it's completely Ok to run `node` without a shell (directly from a Windows app or via `❖Win + r`).
Then there's the whole  `Git for Windows` === `Git Bash` === `MSYS` ~== `MinGW` naming kerfuffle...
And as for terminal emulators, AFAIK `winpty` is only needed when using `mintty`. I don't know what happens when running `MSYS Bash` in `Console2` or `ComEmu`...

tl;dr IMHO this should go in a different section with a simpler explanation of why `winpty` is needed and and which scenarios, and omit the "Officially supported" sentence since that's hard to define, and the sentence above is probably not the most correct approximation (for instance we CI on headless `MSYS Bash` but via `python` that traps the `stdio` file handlers).